--- boost2/boost/python/detail/preprocessor.hpp	2004-07-26 02:32:07.000000000 +0200
+++ boost/boost/python/detail/preprocessor.hpp	2008-01-27 02:07:13.000000000 +0100
@@ -26,7 +26,7 @@
 # endif
 
 # ifndef BOOST_PYTHON_MAX_ARITY
-#  define BOOST_PYTHON_MAX_ARITY 15
+#  define BOOST_PYTHON_MAX_ARITY 19
 # endif
 
 # ifndef BOOST_PYTHON_MAX_BASES
Sólo en boost: .bootstrap.patched
diff -ru boost2/libs/python/build/Jamfile.v2 boost/libs/python/build/Jamfile.v2
--- boost2/libs/python/build/Jamfile.v2	2007-06-08 20:30:43.000000000 +0200
+++ boost/libs/python/build/Jamfile.v2	2008-01-27 01:41:04.000000000 +0100
@@ -58,6 +58,8 @@
     wrapper.cpp
     import.cpp
     exec.cpp
+    indexing/indexing_slice.cpp
+    indexing/python_iterator.cpp
     :   # requirements
         <link>static:<define>BOOST_PYTHON_STATIC_LIB 
         <define>BOOST_PYTHON_SOURCE
diff -ru boost2/tools/jam/src/build.jam boost/tools/jam/src/build.jam
--- boost2/tools/jam/src/build.jam	2006-07-23 18:24:21.000000000 +0200
+++ boost/tools/jam/src/build.jam	2008-01-27 01:39:39.000000000 +0100
@@ -152,7 +152,7 @@
     : -L$(--python-lib[1]) -l$(--python-lib[2]) ;
 ## GCC 2.x, 3.x, 4.x
 toolset gcc gcc : "-o " : -D
-    : -pedantic
+    : -pedantic -fno-strict-aliasing
     [ opt --release : [ opt --symbols : -g : -s ] -O3 ]
     [ opt --debug : -g -O0 -fno-inline ]
     -I$(--python-include) -Wno-long-long
