diff -Naur boost_1_35_0/boost/python/detail/preprocessor.hpp boost/boost/python/detail/preprocessor.hpp
--- boost_1_35_0/boost/python/detail/preprocessor.hpp	2004-07-26 02:32:12.000000000 +0200
+++ boost/boost/python/detail/preprocessor.hpp	2008-07-28 14:58:50.000000000 +0200
@@ -26,7 +26,7 @@
 # endif
 
 # ifndef BOOST_PYTHON_MAX_ARITY
-#  define BOOST_PYTHON_MAX_ARITY 15
+#  define BOOST_PYTHON_MAX_ARITY 19
 # endif
 
 # ifndef BOOST_PYTHON_MAX_BASES
diff -Naur boost_1_35_0/libs/python/build/Jamfile.v2 boost/libs/python/build/Jamfile.v2
--- boost_1_35_0/libs/python/build/Jamfile.v2	2007-11-25 19:38:02.000000000 +0100
+++ boost/libs/python/build/Jamfile.v2	2008-07-28 15:02:01.000000000 +0200
@@ -60,6 +60,9 @@
     import.cpp
     exec.cpp
     object/function_doc_signature.cpp
+    indexing/indexing_slice.cpp
+    indexing/python_iterator.cpp    
+
     :   # requirements
         <link>static:<define>BOOST_PYTHON_STATIC_LIB 
         <define>BOOST_PYTHON_SOURCE
