diff -Naur CELayoutEditor.orig/src/CELayoutEditor.cpp CELayoutEditor/src/CELayoutEditor.cpp
--- CELayoutEditor.orig/src/CELayoutEditor.cpp	2009-06-10 14:58:24.000000000 +0200
+++ CELayoutEditor/src/CELayoutEditor.cpp	2009-06-10 15:03:39.000000000 +0200
@@ -248,7 +248,7 @@
 
     // Assign the log filename
     // NOTE: The file will be output to the current directory (where the executable is located)
-    CEGUI::Logger::getSingleton().setLogFilename("CELayoutEditor.log", false);
+    CEGUI::Logger::getSingleton().setLogFilename("log/CELayoutEditor.log", false);
 
     // Set informative logging level
     CEGUI::Logger::getSingleton().setLoggingLevel (CEGUI::Informative);
@@ -609,8 +609,8 @@
         report->AddFile(wxGetCwd() + wxT("\\CELayoutEditor.log"), wxT("CELayoutEditor log file"));
         report->AddFile(wxGetCwd() + wxT("\\CELayoutEditor.ini"), wxT("CELayoutEditor configuration file"));
     #else
-        report->AddFile(wxGetCwd() + wxT("/CELayoutEditor.log"), wxT("CELayoutEditor log file"));
-        report->AddFile(wxGetCwd() + wxT("/CELayoutEditor.ini"), wxT("CELayoutEditor configuration file"));
+        report->AddFile(wxGetCwd() + wxT("/log/CELayoutEditor.log"), wxT("CELayoutEditor log file"));
+        report->AddFile(wxGetCwd() + wxT("/log/CELayoutEditor.ini"), wxT("CELayoutEditor configuration file"));
     #endif // __WINDOWS__
 
     // Display a preview window if so desired to the let the user review the report and 
diff -Naur CELayoutEditor.orig/src/EditorFrame.cpp CELayoutEditor/src/EditorFrame.cpp
--- CELayoutEditor.orig/src/EditorFrame.cpp	2009-06-10 14:58:24.000000000 +0200
+++ CELayoutEditor/src/EditorFrame.cpp	2009-06-10 15:04:42.000000000 +0200
@@ -408,7 +408,8 @@
 void EditorFrame::InitializeCEGUI()
 {
     // Setup support for TGA,JPG and PNG
-    CEGUI::OpenGLRenderer::setDefaultImageCodecName("SILLYImageCodec");
+    //CEGUI::OpenGLRenderer::setDefaultImageCodecName("SILLYImageCodec");
+    CEGUI::OpenGLRenderer::setDefaultImageCodecName("FreeImageImageCodec");
     //CEGUI::OpenGLRenderer::setDefaultImageCodecName("TGAImageCodec");
 
     m_GUIRenderer = new CEGUI::OpenGLRenderer (0);
diff -Naur CELayoutEditor.orig/src/Options.cpp CELayoutEditor/src/Options.cpp
--- CELayoutEditor.orig/src/Options.cpp	2009-06-10 14:58:24.000000000 +0200
+++ CELayoutEditor/src/Options.cpp	2009-06-10 15:05:04.000000000 +0200
@@ -55,7 +55,7 @@
 #define PATH_LAYOUTS            "LayoutsPath" 
 #define SUPPORTED_PROPERTIES    "SupportedProperties"
 
-#define INI_FILE    "CELayoutEditor.ini"
+#define INI_FILE    "config/CELayoutEditor.ini"
 #define INI_SECTION "SETTINGS"
 
 //////////////////////////////////////////////////////////////////////////
--- CELayoutEditor.orig/bootstrap	2009-06-10 14:58:24.000000000 +0200
+++ CELayoutEditor/bootstrap	2009-06-10 15:17:28.000000000 +0200
@@ -1,6 +1,6 @@
-#!/bin/sh
-
-export WANT_AUTOMAKE='1.7'
-export WANT_AUTOCONF_2_5=1
-
-aclocal -I . $ACLOCAL_FLAGS && autoheader && automake --include-deps --add-missing --foreign && autoconf 
+#!/bin/sh
+
+export WANT_AUTOMAKE='1.7'
+export WANT_AUTOCONF_2_5=1
+
+aclocal -I . $ACLOCAL_FLAGS && autoheader && automake --include-deps --add-missing --foreign && autoconf
--- CELayoutEditor.orig/inc/StringHelper.h	2007-12-20 13:19:22.000000000 +0100
+++ CELayoutEditor/inc/StringHelper.h	2009-06-10 18:09:23.000000000 +0200
@@ -131,6 +131,13 @@
         return str;
     }
 
+    static wxString ToWXString(const char str[]) {
+        return str;
+    }
+
+    static CEGUI::String ToCEGUIString(const char str[]) {
+        return str;
+    }
     /** Helper function which returns all array elements in a single comma-separated and
      * newline-terminated string. The entries can themselves be separated by newlines or not.
      * @access public static 
