--- python-ogre.orig/environment.py	2009-06-02 22:47:09.000000000 +0200
+++ python-ogre/environment.py	2009-06-05 20:58:48.000000000 +0200
@@ -673,7 +673,7 @@
         ]
         libs = [boost.lib,  'OgreMain']
         lib_dirs = [Config.LOCAL_LIB]
-        include_dirs = [boost.PATH, Config.PATH_INCLUDE_Ogre]
+        include_dirs = [Config.PATH_INCLUDE_Ogre] #[boost.PATH, Config.PATH_INCLUDE_Ogre]
         CCFLAGS = ' -D__PYTHONOGRE_BUILD_CODE '
 
 
@@ -763,7 +763,7 @@
         libs.append("User32") # needed for static linking
 
     include_dirs = [
-        boost.PATH,
+        #sd7#boost.PATH,
         Config.PATH_INCLUDE_OIS,
     ]
     lib_dirs = [
@@ -931,12 +931,13 @@
             [0, 'echo Then ensure that OgreGUIRender is compiled with this version of CEGUI', '']
         ]
     include_dirs = [
-        boost.PATH,
+        #sd7#boost.PATH,
         Config.PATH_INCLUDE_CEGUI,
-        Config.PATH_CEGUI,
+        #sd7#Config.PATH_CEGUI,
         Config.PATH_INCLUDE_Ogre_CEGUIRenderer,
         Config.PATH_INCLUDE_Ogre,
-        Config.PATH_INCLUDE_Ogre_Dependencies, ## needed as OgreCEGUI uses CEGUI/.. in #includes
+        #sd7#Config.PATH_INCLUDE_Ogre_Dependencies, ## needed as OgreCEGUI uses CEGUI/.. in #includes
+        Config.LOCAL_INCLUDE
     ]
 
     lib_dirs = [
--- python-ogre.orig/PythonOgreConfig_posix.py	2009-06-02 22:47:09.000000000 +0200
+++ python-ogre/PythonOgreConfig_posix.py	2009-06-05 17:37:32.000000000 +0200
@@ -8,9 +8,12 @@
 SDK = False
 
 # the base of the /usr/... dircetory structure that we are using
-ROOT_DIR = os.path.join(BASE_DIR,'root')
-LOCAL_LIB = os.path.join(ROOT_DIR,'usr/lib')
-LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'usr/include')
+#ROOT_DIR = os.path.join(BASE_DIR,'root')
+#LOCAL_LIB = os.path.join(ROOT_DIR,'usr/lib')
+#LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'usr/include')
+ROOT_DIR = os.environ['PREFIX']
+LOCAL_LIB = os.path.join(ROOT_DIR,'lib')
+LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'include')
 
 ## path to the root of your boost dir, should have 'libs', 'boost' subdirectories
 PATH_Boost = None
