--- /home/sd7/development/check_service/python-ogre.new/environment.py	2008-05-04 18:11:23.000000000 +0200
+++ environment.py	2008-05-04 21:47:23.000000000 +0200
@@ -742,10 +742,11 @@
 
     include_dirs = [Config.PATH_Boost
                     ,Config.PATH_INCLUDE_CEGUI
-                    ,Config.PATH_CEGUI
-                    , Config.PATH_INCLUDE_Ogre_CEGUIRenderer
+                    ,Config.LOCAL_INCLUDE
+                    #sd7#,Config.PATH_CEGUI
+                    #sd7#, Config.PATH_INCLUDE_Ogre_CEGUIRenderer
                     , Config.PATH_INCLUDE_Ogre
-                    , Config.PATH_INCLUDE_Ogre_Dependencies ## needed as OgreCEGUI uses CEGUI/.. in #includes
+                    #sd7#, Config.PATH_INCLUDE_Ogre_Dependencies ## needed as OgreCEGUI uses CEGUI/.. in #includes
                     ]
                   
     lib_dirs = [ Config.PATH_LIB_Boost
@@ -1007,7 +1008,7 @@
         CCFLAGS = ' -DNX_LINUX -DLINUX -DNX_DISABLE_FLUIDS '                
     lib_dirs = [Config.PATH_LIB_Boost,
                 Config.PATH_LIB_Ogre_OgreMain,
-                Config.PATH_LIB_NxOgre,
+#                Config.PATH_LIB_NxOgre,
                 Config.PATH_LIB_PhysX
                 ]
     CheckIncludes=[]
--- /home/sd7/development/check_service/python-ogre.new/PythonOgreConfig_posix.py	2008-03-19 16:21:02.000000000 +0100
+++ PythonOgreConfig_posix.py	2008-05-04 21:50:30.000000000 +0200
@@ -11,16 +11,23 @@
 MAC_cflags = ' --gccxml-cxxflags "-DCF_OPEN_SOURCE -D_POSIX_C_SOURCE -isysroot /Developer/SDKs/MacOSX10.4u.sdk"'
 
 # the base of the /usr/... dircetory structure that we are using
-ROOT_DIR = os.path.join(BASE_DIR,'root')
-LOCAL_LIB = os.path.join(ROOT_DIR,'usr/lib')
-LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'usr/include')
+#ROOT_DIR = os.path.join(BASE_DIR,'root')
+#LOCAL_LIB = os.path.join(ROOT_DIR,'usr/lib')
+#LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'usr/include')
+ROOT_DIR = os.environ['PREFIX']
+LOCAL_LIB = os.path.join(ROOT_DIR,'lib')
+LOCAL_INCLUDE = os.path.join(ROOT_DIR, 'include')
 
 ## path to the root of your boost dir, should have 'libs', 'boost' subdirectories
-PATH_Boost = os.path.join(LOCAL_INCLUDE, 'boost-1_34_1')
+PATH_Boost = os.path.join(LOCAL_INCLUDE, 'boost-1_35')
 ## Path to your boost_pythonxxxx lib file
 PATH_LIB_Boost = LOCAL_LIB
 ## and the name of the boost python library
-LIB_Boost = 'libboost_python-gcc41-1_34_1'
+#LIB_Boost = 'libboost_python-gcc41-1_34_1'
+import re
+a,b = os.popen4("gcc -v")
+MY_GCC_VERSION = "".join(re.search('gcc.* ([0-9]*\.[0-9]*)',b.read()).group(1).split("."))
+LIB_Boost = 'libboost_python-gcc%s-1_35_0' %(MY_GCC_VERSION,)
 
 # in Linux we need to code in the Relative path for the library
 RPATH=os.path.join('\\$$ORIGIN',os.pardir,os.pardir, 'lib') #+ '\'' 
